src/opendlv-video-vpx-encoder.cpp

```diff
@@ -64,7 +64,25 @@ int32_t main(int32_t argc, char **argv) {
         const uint32_t BITRATE{(commandlineArguments["bitrate"].size() != 0) ? std::min(std::max(static_cast<uint32_t>(std::stoi(commandlineArguments["bitrate"])), BITRATE_MIN), BITRATE_MAX) : BITRATE_DEFAULT};
         const bool VERBOSE{commandlineArguments.count("verbose") != 0};
         const uint32_t ID{(commandlineArguments["id"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["id"])) : 0};
-
+        const uint32_t THREADS{(commandlineArguments["threads"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["threads"])) : 4};
+        const uint32_t PROFILE{(commandlineArguments["profile"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["profile"])) : 0};
+        const std::string STEREO_MODE{(commandlineArguments["stereo-mode"].size() != 0) ? commandlineArguments["stereo-mode"] : "mono"};
+        const uint32_t LAG_IN_FRAMES{(commandlineArguments["lag-in-frames"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["lag-in-frames"])) : 0};
+        const uint32_t DROP_FRAME{(commandlineArguments["drop-frame"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["drop-frame"])) : 0};
+        const bool RESIZE_ALLOWED{commandlineArguments.count("resize-allowed") != 0};
+        const uint32_t RESIZE_UP{(commandlineArguments["resize-up"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["resize-up"])) : 0};
+        const uint32_t RESIZE_DOWN{(commandlineArguments["resize-down"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["resize-down"])) : 0};
+        const uint32_t END_USAGE{(commandlineArguments["end-usage"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["end-usage"])) : 0};
+        const uint32_t MIN_Q{(commandlineArguments["min-q"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["min-q"])) : 4};
+        const uint32_t UNDERSHOOT_PCT{(commandlineArguments["undershoot-pct"].size() != 0) ? static_cast<uint32_t>(std::stoi(commandlineArguments["undershoot-pct"])) : 0};
```

Comment:
The default value here was 95; why is it set to 0 now?

Reply:
From http://doxygen.db48x.net/mozilla/html/structvpx__codec__enc__cfg.html#ad279fbb0a9fe9395340913161e28651a:

"This value, expressed as a percentage of the target bitrate, describes the target bitrate for easier frames, allowing bits to be saved for harder frames. Set to zero to use the codec default."